easybuild/easyconfigs/p/PIL/PIL-1.1.7-intel-2015b-Python-2.7.10.eb

```diff
@@ -12,9 +12,17 @@ toolchain = {'name': 'intel', 'version': '2015b'}
 source_urls = ['http://effbot.org/downloads/']
 sources = ['Imaging-%(version)s.tar.gz']
 
+patches = ['PIL-find-deps.patch']
+
 pyver = '2.7.10'
 versionsuffix = '-Python-%s' % pyver
-dependencies = [('Python', pyver)]
+dependencies = [
+    ('zlib', '1.2.8'),
+    ('Python', pyver),
+    ('libjpeg-turbo', '1.4.2'),
+# currently not used because of dependency hell
+#   ('freetype', '2.6.1'),
```

Comment:
should be aligned, so intended by 4 spaces:

```python
dependencies = [
     ('zlib', '1.2.8'),
     ('Python', pyver),
     ('libjpeg-turbo', '1.4.2'),
     # currently not used because of dependency hell
     #('freetype', '2.6.1'),
]
```